### PR TITLE
Update tokenlist to include Rugpull Prevention

### PR DESCRIPTION
**Pull request by moboosted**
Update tokenlist to include Rugpull Prevention

### CODE REVIEW

File: pangolin.tokenlist.json [hunk 1]
```diff
@@ -1249,6 +1249,14 @@
       "name": "Wormhole UST",
       "symbol": "UST",
       "logoURI": "https://raw.githubusercontent.com/pangolindex/tokens/main/assets/0xb599c3590F42f8F995ECfa0f85D2980B76862fc1/logo.png"
-    }
+    },
+    {
+      "chainId": 43114,
+      "address": "0x61ecd63e42c27415696e10864d70ecea4aa11289",
```
Comment (bmino) [inline pangolin.tokenlist.json:1255]: Please replace with the checksummed address:
```
0x61eCd63e42C27415696e10864d70ecEA4aA11289
```

File: pangolin.tokenlist.json [hunk 2]
```diff
@@ -1249,6 +1249,14 @@
...
       "logoURI": "https://raw.githubusercontent.com/pangolindex/tokens/main/assets/0xb599c3590F42f8F995ECfa0f85D2980B76862fc1/logo.png"
-    }
+    },
+    {
+      "chainId": 43114,
+      "address": "0x61ecd63e42c27415696e10864d70ecea4aa11289",
+      "decimals": 18,
+      "name": "Rugpull Prevention",
+      "symbol": "RUGPULL",
+      "logoURI": "https://raw.githubusercontent.com/pangolindex/tokens/main/assets/0x61ecd63e42c27415696e10864d70ecea4aa11289/logo.png"
```
Comment (bmino) [inline pangolin.tokenlist.json:1259]: same here with the checksummed address

**Review by bmino — CHANGES_REQUESTED** [review]

File: pangolin.tokenlist.json (hunk 2 above)
Comment (moboosted) [reply pangolin.tokenlist.json:1259]: New pull request added with Rugpull Prevention checksum address

0x61eCd63e42C27415696e10864d70ecEA4aA11289

Comment (moboosted) [reply pangolin.tokenlist.json:1259]: Pull Request #283

**Comment by bmino** [comment]
Closing in favor of #283

**Pull request closed without merging**